test/tag.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+load helpers
+
+
+IMAGE="docker.io/library/alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
+
+function teardown() {
+    cleanup_test
+}
+
+@test "kpod tag with shortname:latest" {
+	run ${KPOD_BINARY} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} tag $IMAGE foobar:latest
```

Comment:
Can you use kpod images to make sure that there's a foobar tag after kpod tag runs?

Reply:
I think the rmi is basically doing that.  If we wanted, we could perform an inspect.

Reply:
I'd just like to have something in there to make sure we know what's going on if the rmi fails - was it because the tag wasn't there, or are the tests in general blowing up

Reply:
Ok, doing an inspect and outputting data for that case.